libs/community/langchain_community/document_loaders/sql_database.py

```diff
@@ -0,0 +1,138 @@
+from typing import Any, Callable, Dict, Iterator, List, Optional, Union
+
+import sqlalchemy as sa
+
+from langchain_community.docstore.document import Document
+from langchain_community.document_loaders.base import BaseLoader
+from langchain_community.utilities.sql_database import SQLDatabase
+
+
+class SQLDatabaseLoader(BaseLoader):
+    """
+    Load documents by querying database tables supported by SQLAlchemy.
+
+    For talking to the database, the document loader uses the `SQLDatabase`
+    utility from the LangChain integration toolkit.
+
+    Each document represents one row of the result.
```

Comment:
Would you be willing to add a ..code-block python example that shows how to use the loader?

It'll populate the API reference with a nice python formatted example:  https://api.python.langchain.com/en/v0.0.349/api_reference.html